components/StripeConnectCard.vue

```diff
@@ -0,0 +1,218 @@
+      />
+    </template>
+
+    <template v-if="isStripeConnectChecked">
+      <div class="flex flex-col gap-[24px]">
+        <div class="flex items-start justify-start gap-[8px] w-full">
+          <URadio
+            v-model="isUsingDefaultAccount"
+            class="w-[50%]"
+            :value="true"
+            :disabled="!!props.shouldDisableSetting"
+          >
+            <template #label>
+              <div>Use my account</div>
+              <div
+                v-if="isDefaultAccountReady"
+                class="flex flex-col items-start"
+              >
+                <div
+                  class="flex flex-col gap-[8px] mt-[12px] px-[6px] py-[4px]"
+                >
+                  <div>✅ Has Stripe Account</div>
+                  <div>{{ ` Email: ${props.stripeConnectStatusWalletMap[props.loginAddress]?.email}` }}</div>
+                </div>
+              </div>
+
+              <span v-else>
+                <UButton
+                  class="mt-2"
+                  label="Create one here"
+                  :to="{ name: 'nft-book-store-user' }"
+                  target="_blank"
+                  variant="outline"
+                />
+              </span>
+            </template>
+          </URadio>
+          <URadio
+            v-model="isUsingDefaultAccount"
+            :value="false"
+            :disabled="!!props.shouldDisableSetting"
+            class="w-[50%]"
+          >
+            <template #label>
+              <span>Use another Stripe Express account</span>
+              <div class="flex flex-col my-[10px]">
+                <UInput
+                  :value="inputWallet"
+                  :color="stripeConnectInputError ? 'rose' : 'white'"
+                  class="font-mon w-full"
+                  placeholder="like1..."
+                  @input="onStripeConnectWalletInput"
+                />
+                <span
+                  v-if="stripeConnectInputError"
+                  class="text-red-700 text-[10px]"
+                >{{ stripeConnectInputError }}</span>
+              </div>
+              <div v-if="isStripeConnectLoading" class="text-center">
+                Loading ...
+              </div>
+              <div
+                v-else-if="
+                  props.stripeConnectStatusWalletMap[inputWallet]
+                "
+                class="flex flex-col gap-[8px] mt-[12px] px-[6px] py-[4px]"
+              >
+                <div
+                  v-if="isInputAccountReady"
+                >
+                  <span> ✅ Has Stripe Account</span><br>
+                  <span
+                    v-if="
+                      props.stripeConnectStatusWalletMap[inputWallet]
+                        ?.email
+                    "
+                  >{{
+                    ` Email: ${props.stripeConnectStatusWalletMap[inputWallet]?.email}`
+                  }}</span>
+                </div>
```

Comment:
Try to make use of Nuxt UI component like `<UAlert/>`

Reply:
[💄Adjust style & wording](https://github.com/likecoin/nft-book-press/pull/139/commits/a6b3d1fc776e8e2e295dfe49b9e4e79b3f745315)

Reply:
[💄Update radio group style](https://github.com/likecoin/nft-book-press/pull/139/commits/d591169181cccc1b7d476837e23aadce8886512a)